### PR TITLE
[SDK-2612] [Beta] Quickstart for PHP SDK v8.0

### PR DESCRIPTION
**Pull request by evansims**
This is a major rewrite of the PHP web app quickstart:

- Adds support for the SDK v8.0 release
- Rewrote code base to better reflect modern PHP apps
- Rewrote Docker support
- Updated documentation
- Moved project to the root level of the project, as the numbered subfolder approach won't be necessary in the future: this project supports hot-swappable modules to demonstrate extra functionality without having to have redundant copies of the entire project duplicated throughout subfolders.

You can run the application through Docker via:
```sh
composer run docker
```

### CODE REVIEW

**Comment by evansims** [comment]
NOTE: Until the SDK 8.0 beta release is formally made, this QS points to the 8.x-dev branch. This will be updated to point to the beta release before merge.

**Comment by evansims** [comment]
Switching the base branch on this from `master` to a new `8.x` branch to house this during the beta phase, as the new file structure won't allow subfolders.

File: .env.example [hunk 1]
```diff
@@ -1,11 +1,26 @@
 # Your Auth0 application's Client ID
 AUTH0_CLIENT_ID='YOUR_AUTH0_CLIENT_ID'
```
Comment (davidpatrick) [inline .env.example:2]: You can use variable replacement here with `AUTH0_CLIENT_ID={CLIENT_ID}`

File: .env.example [hunk 2]
```diff
@@ -1,11 +1,26 @@
...
 
-# The url of your Auth0 tenant domain
+# The URL of your Auth0 tenant domain
 AUTH0_DOMAIN='https://YOUR_AUTH0_DOMAIN.auth0.com'
```
Comment (davidpatrick) [inline .env.example:5]: Variable replacement `AUTH0_DOMAIN=https://{DOMAIN}`

File: .env.example [hunk 3]
```diff
@@ -1,11 +1,26 @@
...
 
 # Your Auth0 application's Client Secret
 AUTH0_CLIENT_SECRET='YOUR_AUTH0_CLIENT_SECRET'
```
Comment (davidpatrick) [inline .env.example:8]: Variable replacement `AUTH0_CLIENT_SECRET={CLIENT_SECRET}`

Comment (evansims) [reply .env.example:8]: Didn't realize about these, thanks!

**Review by davidpatrick — APPROVED** [review]

**Pull request merged**